scalameta/tokenizers/shared/src/main/scala/scala/meta/internal/tokenizers/ScalametaTokenizer.scala

```diff
@@ -324,6 +329,9 @@ class ScalametaTokenizer(input: Input, dialect: Dialect) {
     tokens.toArray(underlying)
     Tokens(underlying, 0, underlying.length)
   }
+
+  private def syntaxError(startPos: Int, endPos: Int, message: String): Nothing =
```

Comment:
It's a bit weird to have a syntax error in the tokenizer, since it doesn't deal with syntax. This would still be better than the previous implementation, but I wonder if we could just skip the interpolation if it doesn't have an end? What do you think?

Reply:
as it is kind of messy recursive flow it is not easy to just simply stop. 
But if it is okay to skip not fully closed token, I propose v0.2 in my commit, WDYT?

Reply:
aaaaand we are hitting on JS https://github.com/scala/bug/issues/11920
Cannot throw AIOOB as it causes UndefinedBehaviour and you cannot catch it ...

Reply:
aaaaand worked around like a boss :sunglasses: